### PR TITLE
Oleksandr_Tsapin_HW7

### PR DESCRIPTION
**Pull request by OleksandrTsapin**

### CODE REVIEW

File: tsapin/HW7/task1.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    if i not in num_and_arithm_oper:
+        char_string_result += i
+    elif char_string.count(i) == 1:
+        char_string_result += i
+    else:
+        if i not in char_string_result:
+             char_string_result += i
+
+print('String given: ' + char_string,
+      '\nResult: ' + char_string_result)
```
Comment (katemalash) [inline tsapin/HW7/task1.py:19]: Good Way.

File: tsapin/HW7/task2.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+for i in str2:
+    if i not in num:
+        set_str2.add(i)
+
+intersection = set_str1.intersection(set_str2)
+sum_of_difference = (set_str1.difference(set_str2)).union(set_str2.difference(set_str1))
+
+print('Letters in both arrays: ', sorted(intersection),
+      '\nLetters in 1st & 2nd arrays separately: ', sorted(sum_of_difference))
```
Comment (katemalash) [inline tsapin/HW7/task2.py:24]: Nice Decision

File: tsapin/HW7/task3.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+while True:
+    user_input = input("\nEnter the city (type 'exit' to stop). \n--> ")
+    if user_input == 'exit':
+        print("Bye!")
+        break
+    else:
+        for i in major_cities:
+            if user_input in major_cities[i]:
+                print(i)
```
Comment (katemalash) [inline tsapin/HW7/task3.py:22]: Nice.

File: tsapin/HW7/task4.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# by one or more spaces. Create a dictionary in which the keys are the words from the sentence, and the 
+# values ​​- the number of it's occurrences in the sentence.
+
+user_text = input("Your text: ").split()
+dictionary = {}
+
+for i in user_text:
+    dictionary[i] = user_text.count(i)
+
+print("-"*42,
```
Comment (katemalash) [inline tsapin/HW7/task4.py:12]: Why 42?)

File: tsapin/HW7/task4.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+      "\nWord : Number of occurrences\n",dictionary)
```
Comment (katemalash) [inline tsapin/HW7/task4.py:13]: Nice Decision.

**Review by katemalash — APPROVED** [review]
Good Job. You can merge pull-request now.

File: tsapin/HW7/task4.py (hunk 4 above)
Comment (OleksandrTsapin) [reply tsapin/HW7/task4.py:12]: "Answer to the Ultimate Question of Life, the Universe, and Everything" ;)

**Pull request merged**